.github/workflows/vscode-extension.yml

```diff
@@ -45,20 +45,35 @@ jobs:
         with:
           node-version: 16.16.x
 
+      - name: Install Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: "1.23.4"
+
       - name: Attempt to rebuild
         run: npm rebuild | true
       
       - name: Install Dependencies
         run: npm ci
 
       - name: Compile packages
-        run: npm run build-all
+        run: |
+          echo "$(pwd)" >> $GITHUB_PATH
+          npm run build-all
+
+      - name: Make taq binary globally visible
```

Comment:
If the directory output from `pwd` is the same as the directory output from which taq in this step are the same I think this step can be removed since it will be accessible if the above command works

Reply:
Fixed.